controllers/errorHandlingcontrollers.js

```diff
@@ -5,7 +5,7 @@ function handle404NonExistentPaths(req, res, next) {
 function handlePSQL400err(err, req, res, next) {
   const psql400errors = ["22P02", "23502"];
 
-  if (psql400errors.includes(err.code)) {
+  if (psql400errors.includes(err.code) || err === "Bad request") {
```

Comment:
Change this to deal with the codes rather than messages from your model

Reply:
thanks August :) all the changes have been added.